blueocean-dashboard/src/main/js/jenkins-js-extension.yaml

```diff
@@ -41,3 +41,13 @@ extensions:
 # Trends
   - component: components/trends/StageDurationChart
     extensionPoint: jenkins.pipeline.trends
+# Actions
```

Comment:
Register default run details components (changes, tests, and artifacts) as extensions of the `jenkins.run.actions` extension point. External plugins that want to extend this extension point should similarly register components in their`jenkins-js-extension.yaml` file.